Sources/XMTPiOS/Client.swift

```diff
@@ -100,23 +101,40 @@ public final class Client: Sendable {
 	}
 
 	static func create(account: SigningKey, apiClient: ApiClient, options: ClientOptions? = nil) async throws -> Client {
-		let privateKeyBundleV1 = try await loadOrCreateKeys(for: account, apiClient: apiClient, options: options)
+		let (privateKeyBundleV1, source) = try await loadOrCreateKeys(for: account, apiClient: apiClient, options: options)
+
+		let dbURL = URL.documentsDirectory.appendingPathComponent("xmtp-\(account.address).db3")
+		let v3Client = try await LibXMTP.createClient(
+			logger: XMTPLogger(),
+			host: GRPCApiClient.envToUrl(env: apiClient.environment),
+			isSecure: apiClient.environment != .local,
+			db: dbURL.path,
+			encryptionKey: nil,
+			accountAddress: account.address,
+			legacyIdentitySource: source,
+			legacySignedPrivateKeyProto: nil
+		)
+
+		guard let textToSign = v3Client.textToSign() else {
+			throw ClientError.creationError("no text to sign")
```

Comment:
It's a totally okay scenario to not have text to sign. That actually means you already have V3 keys. We shouldn't error here.

Reply:
Good to know, updated.